vignettes/decorate-multiple-modules-outputs.Rmd

```diff
@@ -0,0 +1,203 @@
+---
+title: "Customizing Multiple Modules Outputs"
+author: "NEST CoreDev"
+output:
+  rmarkdown::html_vignette:
+    toc: true
+vignette: >
+  %\VignetteIndexEntry{Customizing Multiple Modules Outputs}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+Below document contains example code presenting how you can change multiple objects inside one module.
+This is an extension of `Customizing Module Output` vignette.
+
+```{r}
+library(ggplot2)
+tm_decorated_plot <- function(
+    label = "module",
+    decorator = list(
+      plot_1 = teal_transform_module(),
+      plot_2 = teal_transform_module()
+    )) {
+  # decorator: teal_transform_module, language, function
+  decorator_objs <- list(
+    plot_1 = decorate_teal_data(x = decorator[["plot_1"]], output_name = "plot_1"),
+    plot_2 = decorate_teal_data(x = decorator[["plot_2"]], output_name = "plot_2")
```

Comment:
This function is missing `decorate_teal_data`

Reply:
Yup, this feature is abandoned and we decide to no go this way. See the discussion [2. Decorator as a function](https://github.com/insightsengineering/teal/issues/1384#issuecomment-2416945192) and to understand the nuances affecting decision you can read everything from the first to the last comment. The key takeaway is that `decorate_teal_data` call was needed in the earlier stage of the research, where we considered a case where app-developer doesn't know the name of the internal object in the teal_module. We agreed that one should specify working teal_transform_module, so this step isn't needed.